### PR TITLE
Mike Grillo

### PR DESCRIPTION
**Pull request by Mvgrillo1357**
this is my first file

### CODE REVIEW

File: mikeGrillo.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jan  3 14:05:29 2020
+
+@author: michael
+"""
+
+def grillom(a:int) -> str:
```
Comment (kylestormcloud) [inline mikeGrillo.py]: Takes in an integer

File: mikeGrillo.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+###
+#this is my function
+
+###
+    if a==1:
+        return "this is my first string"
+    elif a==2:
+        return "this is my second string"
+    else:
+        return "this is my third string"
```
Comment (kylestormcloud) [inline mikeGrillo.py:20]: Returns one of three strings

**Review by kylestormcloud — APPROVED** [review]
You are approved!

**Review by srwight — CHANGES_REQUESTED** [review]
When you do a docstring, use """ instead of ###

**Review by Maraudier — APPROVED** [review]
I am Matt Davis and I approve this message

File: mikeGrillo.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jan  3 14:05:29 2020
+
+@author: michael
+"""
+
+def grillom(a:int) -> str:
+
```
Comment (srwight) [inline mikeGrillo.py:10]: docstring should go here. 
""" This is my dosctring"""

**Comment by srwight** [comment]
superceded

**Pull request closed without merging**